src/store/Woofs/slice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  Woofs: "some initial value",
```

Comment:
there will be multiple woofs in here, so this initial state needs to be an array.

The `woofs` key is already in the redux store, so just the value should be enough, I think.

Finally, in general, keep Object keys lower case too, please. Captitals or CamelCase are only used for classes, like Array or Car or Post, or.. Woof, actually. But not Woof`s`.

To conclude, initial state should be: 

```js
const initialState = [];
```

Hope this makes sense.

Reply:
Clear!